content/blog/clj-rq-v0.2-release.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Our lib `clj-rq` now generates itself dynamically"
+date: 2024-08-23T01:00:00+10:00
+tags: ["foss", "clj-rq", "release"]
+images: TODO
```

Comment:
Add images. Reminding myself.

Reply:
what do you think?

![e5e61e734065bf1b98be7e08213caf0ce8be31b8](https://github.com/user-attachments/assets/25332604-b0de-4a7c-a8c7-25dd3c13c18b)

Reply:
very good!